packages/imperative/src/config/src/Config.ts

```diff
@@ -147,10 +147,10 @@ export class Config {
         opts = opts || {};
 
         // Create the basic empty configuration
-        const myNewConfig = new Config(opts);
+        const myNewConfig = new Config();
         myNewConfig.mApp = app;
         myNewConfig.mActive = { user: false, global: false };
-        myNewConfig.mVault = opts.vault;
+        myNewConfig.mVault = opts?.vault;
```

Comment:
Since opts is defined on L147, is the `?` necessary? Just curious, since Coverity tends to be rather strict.

Reply:
Makes sense to remove... the other option would've been to remove the assignment a few lines above 😋